v2/internal/reconcilers/error_classifier.go

```diff
@@ -86,10 +88,20 @@ func classifyInnerCloudError(err *genericarmclient.ErrorResponse) CloudErrorClas
 		"ResourceQuotaExceeded",
 		"SubscriptionNotRegistered":
 		return CloudErrorRetryable
+	case "Conflict":
+		// Conflict should really be fatal, but there are some
+		// services that include a message that we should try again
+		// later. In that case classify it as retryable.
```

Comment:
This is one of those things that _works_, but really leaves a sour taste in the back of the throat.

Reply:
Yeah definitely - once we change things allow customising classification at the resource level this will be a bit better contained.